### DISCUSSION TITLE
Eclipse support?

### DISCUSSION THREAD

**Discussion by phraktle** [Randgalt/record-builder#51] [General]
Hi,

Are there any instruction on getting the annotation processor working within Eclipse?

Specifically,  I've tried adding `record-builder` using the [gradle processors plugin](https://github.com/palantir/gradle-processors), which appears to add the appropriate configuration (record-builder-core, record-builder-process and javapoet on the APT factory path), but I'm still getting an `Unable to load annotation processor factory 'io.soabase.recordbuilder.processor.RecordBuilderProcessor' for project` error in Eclipse (which I was so far unable to diagnose further). 

build.gradle.kts:
```
plugins {
    eclipse
    id("org.inferred.processors")
    ...
}

dependencies {
    annotationProcessor("io.soabase.record-builder:record-builder-processor:22")
    compileOnly("io.soabase.record-builder:record-builder-core:22")
    ...
}
```

**Comment by Randgalt** [comment]
I apologize, I'm not an Eclipse user. I'll try to find some time to test with it. But, I'd appreciate it if you would post back with anything you find.
